PeerioTechnologies/peerio-desktop
# fix: jump to bottom button prevents scrolling [ch11618]

Hide jump to bottom button when getting scroll wheel events on it, and
show it back after 1 second (debounced).

#### Relevant info and issue/PR links 
 
#### Testing instructions  

1. Scroll until jump to bottom button appears.
2. Scroll with mouse wheel/touchpad over this button -- the button should disappear and the message list should scroll. The button will reappear after a short time when scrolling finishes.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
